test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -18,8 +18,7 @@ package zio.test
 
 import scala.reflect.ClassTag
 
-import zio.Cause
-import zio.Exit
+import zio.{ Cause, Exit, ZIO }
```

Comment:
I suppose we can add assertions on effects directly now, e.g. `failsM(zio)` and the like...